source/includes/json/_local_details.md

```diff
@@ -59,9 +47,23 @@ It can be used alongside the `/api/local/points` endpoint; the criteria for incl
 
 #### Query Parameters
 
-Parameter | Default | Description
---------- | ------- | -----------
-page      | 1       | (optional) Which page of results to fetch
-per_page  | 10      | (optional) How many results, maximum, should be included on each page
+All parameters are optional.
+
+Parameter                | Default                | Description
+---------                | -------                | -----------
+filter[boundary_box]     | null                   | If there is a location filter, bounding box to use for geocoding the location query.
+filter[calendars]        | null (no filter)       | List of calendar slugs. If specified, only events in those calendars will be returned.
+filter[event_types]      | null (no filter)       | List of event type IDs. If specified, only events of those types will be returned.
+filter[geographies]      | null (no filter)       | List of geographic shape IDs. If specified, only events/groups whose locations fall within those geographies will be returned.
```

Comment:
Do we expose the geographies' ID anywhere (other than on full table exports)? Maybe we should leave this option undocumented to push people to use the `geography_slugs` instead

Reply:
Good question! It looks like we do use geography IDs in some URLs, although of course everything about geographies in the org admin UI is a little bit hidden.